examples/sid_visualization/README.md

```diff
@@ -0,0 +1,135 @@
+# SID Visualization Example
+
+## Prerequisites
+
+To run the demo you will need the following:
+- Docker
+- NodeJS
+  - See the installation guide [here](https://nodejs.org/en/download/)
+- `yarn`
+  - Once NodeJS is installed with NPM, run `npm install --global yarn` to install yarn
+- `docker-compose` (Tested with version 1.29)
+
+## Setup
+
+To run this demo, ensure all submodules are checked out
+
+```bash
+git submodule update --init --recursive
```

Comment:
It would be good if all of these code blocks could be easily copied with a click so the user doesn't have to drag copy then paste.

Reply:
i'll move each command into it's own code block so you can just click copy to clipboard button.